.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+
+contact_links:
+  - name: Ask a question
+    url: https://github.com/wprig/wprig/discussions/new
+    about: Ask and discuss questions with other Bootstrap community members
```

Comment:
Should we change line 5 to with other WPRig community members instead

Reply:
I think we should have Ask A Questions go specifically to eh Q/A area of the discussion board.